ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+<!-- Thanks for opening an issue. To help us replicate your issue can you please answer the following questions. -->
+
+1. What operating system do you use?
+
+2. What version of Vagrant are you running?
+
+3. Are you using VirtualBox or VMWare and which version are you using?
+
+4. Did you have a custom yaml file? If so what does it contain? (Please paste in your customc .yaml file)
```

Comment:
1. `Do you have...` rather than `Did you have...`
2. `custom.yaml` in backticks rather than `customc .yaml` without backticks